subprojects/soak/src/integTest/groovy/org/gradle/vfs/VirtualFileSystemRetentionSoakTest.groovy

```diff
@@ -92,9 +92,7 @@ class VirtualFileSystemRetentionSoakTest extends DaemonIntegrationSpec implement
             boolean overflowDetected = detectOverflow(daemon, endOfDaemonLog)
             int expectedNumberOfRetainedFiles = retainedFilesInLastBuild - numberOfChangesBetweenBuilds
```

Comment:
Only calculate this if `!overflowDetected`.

Reply:
Done.